python/ql/test/library-tests/web/django/Sinks.expected

```diff
@@ -1,6 +1,8 @@
 | sql.py:13 | Str | externally controlled string |
 | sql.py:14 | Str | externally controlled string |
 | sql.py:17 | BinaryExpr | externally controlled string |
+| sql.py:17 | Str | externally controlled string |
```

Comment:
Line 17 `cursor.execute("SELECT * FROM users WHERE username = '%s'" % username)`

so does this mean that now we treat the string `"SELECT * FROM users WHERE username = '%s'"` as a _externally controlled string_? and isn't that a bit strange? :confused:

Reply:
Hmm... That _does_ sound a bit fishy. I'll investigate.

Reply:
Ah, what this is _actually_ saying is that the node in question is a _sink_ of that particular kind of taint. That's slightly less worrisome, although that now makes me wonder whether `username` should also count as a sink, separately (probably not).